### PR TITLE
Changes to use helixApi for EventHub.

### PR DESCRIPTION
**Pull request by venkat-raman251**
@ChadNedzlek  Please have a look.
I tested out build.cmd with and without the api as well as the error when neither params are present in the rsp

### CODE REVIEW

File: src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets [hunk 1]
```diff
@@ -70,12 +72,17 @@
...
-    <Error Condition="'$(EventHubPath)' == ''" Text="Missing required property EventHubPath." />
-    <Error Condition="'$(EventHubSharedAccessKey)' == ''" Text="Missing required property EventHubSharedAccessKey." />
-    <Error Condition="'$(EventHubSharedAccessKeyName)' == ''" Text="Missing required property EventHubSharedAccessKeyName." />
     <Error Condition="'$(BuildIsOfficial)' == 'true' and '$(BuildIsOfficialEventHubPath)' == ''" Text="Missing required property BuildIsOfficialEventHubPath." />
     <Error Condition="'$(BuildIsOfficial)' == 'true' and '$(BuildIsOfficialEventHubSharedAccessKeyName)' == ''" Text="Missing required property BuildIsOfficialEventHubSharedAccessKeyName." />
     <Error Condition="'$(BuildIsOfficial)' == 'true' and '$(BuildIsOfficialEventHubSharedAccessKey)' == ''" Text="Missing required property BuildIsOfficialEventHubSharedAccessKey." />
+    <!-- TODO: Once we all users are no longer setting EventHubInfoMissing, we can remove this message, and the '$(EventHubInfoMissing)' == '' condition from the error following -->
+    <Message Text="venkat : BuildCompleteConnection = $(BuildCompleteConnection)"/>
```
Comment (ChadNedzlek) [inline src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets]: This line shouldn't be here...

File: src/Microsoft.DotNet.Build.CloudTestTasks/SendToHelix.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            Log.LogMessage(MessageImportance.Low, "Event json is ", EventDataPath);
+
+            using (HttpClient client = new HttpClient())
+            {
+                int retryCount = 15;
+                while (true)
+                {
+                    HttpResponseMessage response;
+                    client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+                    Log.LogMessage("venkat changes log msg");
```
Comment (ChadNedzlek) [inline src/Microsoft.DotNet.Build.CloudTestTasks/SendToHelix.cs]: Another sneaky line that should probably get deleted.

**Comment by venkat-raman251** [comment]
@ChadNedzlek  : Updated.

**Comment by venkat-raman251** [comment]
@ChadNedzlek : Sorry, I missed the .csproj in the commit. Added it now.

**Pull request merged**